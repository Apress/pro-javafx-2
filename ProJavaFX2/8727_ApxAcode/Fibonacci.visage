/*
 * @test
 * @run
 */

function fib(n: Integer): Integer {
  if (n <= 0) then 0 else
  if (n == 1) then 1 else
  fib(n - 1) + fib(n - 2)
}

println("fib(6) = {fib(6)}"); 
