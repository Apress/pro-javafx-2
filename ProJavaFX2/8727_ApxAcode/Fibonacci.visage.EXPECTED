fib(6) = 8
